src/main/resources/db/changelog/create-table-books.yml

```
@@ -0,0 +1,48 @@
+databaseChangeLog:
+        - changeSet:
+                  id: "1"
+                  author: "developer"
+                  changes:
+                          - createTable:
+                                    tableName: "books"
+                                    columns:
+                                            - column:
+                                                      name: "id"
```

remove double quotation everywhere
```suggestion
                                                      name: id
```